src/documents/tutorial-iconFonts/HowToCreateAndUseFontIcons.md

```diff
@@ -301,26 +342,23 @@ on Windows and Mac OS X 10.6+.
 
 ![Unicode character map](../images/Unicode-charmap.png)
 
-#### Obtaining Unicode in Mac OS X 10.6 or Later
+## What about SVG Icons
 
-1. Enable Special Characters support as documented in this Apple Knowledge Base article:
-   [OS X Lion: Enter special characters and symbols](http://support.apple.com/kb/ph3871)
-2. Install the custom TTF font to the OS (usually double-click, then "Install").
-3. Launch the Font Book application.
-4. Also launch the Notes application.
-5. In Font Book, locate the custom font under Collection > All Fonts. Select it. The custom font should now appear in
-   the window.
-6. In Font Book, select / highlight the characters from the custom font. Select Copy (Command+C or right-click -> Copy).
-7. In Notes, Paste the text (Command+V or right-click -> Paste). The custom font should appear in Notepad.
-8. In Notes, select the pasted text. Then open the "Edit" menu an select "Special Characters".
-9. Now a "Characters" window appears with "Search" results containing each character selected in the previous step.
-10. Select an icon in the "Character" panel. The Unicode will appear in the right column.
+SVG icons can be used as an alternative to icons supplied through an icon font. Below is a quick summary of the pros and
+cons of SVG Icons. For more information on SVG Icons see the
+[Research SVG Icons](https://wiki.fluidproject.org/display/fluid/Research+SVG+Icons) wiki page.
 
-![Mac OS X Font Book](../images/mac-font-book.png)
+### Benefits of SVG Icons
 
-Above: An image showing the Mac OS X Font Book application with the custom characters highlighted.
+* Can use multiple colours
+* Can support complex icons including multiple colours
+  * although it increases the complexity to style with CSS
+* Don't have to compile into a font
 
-![Mac OS X Characters window](../images/mac-characters-window.png)
+### Drawbacks of SVG Icons
 
-Above: An image showing the Mac OS X Characters window. The Unicode value for a custom character appears in the right
-most column.
+* Less browser support, although modern browsers should handle most features
+* Many different ways to add to a page, but only those that allow access to the SVG markup will permit styling via CSS
+* Styling multi-coloured SVGs can be complex and may not always be supported
```

Comment:
It's not really necessary to say that styling SVGs with multiple colours is complex.

Can you give an example where styling an SVG with multiple colours using CSS isn't supported by a browser that supports SVGs?